java/src/jmri/jmrix/AbstractMRNodeTrafficController.java

```diff
@@ -103,9 +105,11 @@ public synchronized AbstractNode getNode(int index) {
     }
 
     /**
-     * Public method to identify a SerialNode from its node address Note: 'addr'
-     * is the node address, numbered from 0. Returns 'null' if a SerialNode with
-     * the specified address was not found
+     * Public method to identify a SerialNode from its node address.
```

Comment:
Please don't use "Public method to verb ...", but simply use "Verb ...". There have been instances where the scope in the doc does not match the scope of the method in code, and hunting down which is correct can be problematic. Its better to have a single source for that information (the code) in that case.

Reply:
Tend to rewrite my own Javadoc accordingly, will change these